### ISSUE TITLE
1.8.2 typescript-react-apollo doesn't properly implement the typesPrefix option.

### ISSUE THREAD

**Issue by Rachnerd** [dotansimha/graphql-code-generator#2826]
**Describe the bug**
`@graphql-codegen/typescript-react-apollo` doesn't respect capitalized type prefixes anymore since 1.8.2. This results in a mismatch with the other 1.8.2 plugins that do properly implement capitalized type prefixes.

**To Reproduce**
Install `@graphql-codegen/typescript-react-apollo@1.8.2` and generate a hook with typesPrefix set to `GQL`. Results in:
`return ApolloReactHooks.useQuery<GqlMyQuery`. 

instead of the expected:
`return ApolloReactHooks.useQuery<GQLMyQuery`.

My `codegen.yml` config file:

```yml
schema:
  - schema-location
documents: "./src/**/*.graphql"
overwrite: true
generates:
  ./.generated/fragmentTypes.json:
    plugins:
      - fragment-matcher
  ./.generated/gql.model.tsx:
    plugins:
      - typescript
      - typescript-operations
      - typescript-react-apollo
    config:
      scalars:
        Time: string
      withHooks: true
      withHOC: false
      hooksImportFrom: "@apollo/react-hooks"
      avoidOptionals: true
      typesPrefix: GQL
  ./.generated/schema.graphql:
    - schema-ast
```

**Expected behavior**
All plugins should implement the typesPrefix config the same way.

**Environment:**

- OS: Linux Ubuntu 19.04
- `@graphql-codegen/typescript-react-apollo`: 1.8.2
- NodeJS: v10.14.2

**Comment by xiaoyu-tamu** [comment]
Another issues related to 1.8.2 #2742

**Comment by dotansimha** [comment]
I think I managed to reproduce and fix this in: https://github.com/dotansimha/graphql-code-generator/pull/2862 


@Rachnerd @xiaoyu-tamu Can you please try this alpha? `1.8.3-alpha-57c54418.34`

**Comment by Rachnerd** [comment]
@dotansimha `1.8.3-alpha-57c54418.34` fixes the issue. Thanks!

**Comment by xiaoyu-tamu** [comment]
@dotansimha  it works. Thank you!

**Comment by dotansimha** [comment]
Fixed in 1.8.3
